internal/k8s/client/client.go

```diff
@@ -49,13 +54,21 @@ type (
 	ListOption         = cli.ListOption
 	CreateOption       = cli.CreateOption
 	CreateOptions      = cli.CreateOptions
+	GetOption          = cli.GetOption
+	GetOptions         = cli.GetOptions
 	UpdateOptions      = cli.UpdateOptions
 	MatchingLabels     = cli.MatchingLabels
 	InNamespace        = cli.InNamespace
 	VolumeSnapshot     = snapshotv1.VolumeSnapshot
+	Pod                = corev1.Pod
 	Deployment         = appsv1.Deployment
 	DeploymentList     = appsv1.DeploymentList
 	ObjectMeta         = metav1.ObjectMeta
+	EnvVar             = corev1.EnvVar
+	Job                = batchv1.Job
+	JobList            = batchv1.JobList
```

Comment:
Please explain why define `Job` and `JobList` here.
`Job` is already at defined `internal/k8s/job/job`